umake/frameworks/ide.py

```diff
@@ -77,7 +77,6 @@ def __init__(self, *args, **kwargs):
             kwargs["required_files_path"] = current_required_files_path
         download_page = 'https://www.eclipse.org/downloads/'
         kwargs["download_page"] = download_page
-        kwargs["exec_rel_path"] = "eclipse"
```

Comment:
It should be removed..
I made the changes on the wrong branch, so it might be that I forgot some when creating a new one